### ISSUE TITLE
Custom Emoji issue

### ISSUE THREAD

**Issue by SINE** [peterthehan/discord-reaction-role-bot#5]
Hello!

First off, great project, thanks a lot as it really takes a lot work off people's bottoms.
Anyways, to the issue I'm having.

Everything properly set up, if I start using a custom emoji, that is being added to the message as expected.

However, if I click on the emoji, the role isn't being assigned properly - in contrary to a utf-8 emoji. The reaction is being removed instantly as expected, so the bot "works with it", however it doesn't identify that the reaction was clicked.

I'm assuming it is a bug but didn't find the source yet. Maybe you'll have an idea. I'm assuming it could be an issue with identifying what emoji was clicked since it expects :emojiname:emojid, contrary to documentation.

Might be worth checking further

**Comment by peterthehan** [comment]
Hey @SINE, can you post your config?

Also, do you mind providing more context on this custom emoji? Does it exist in the server the bot is in?

Also to rule out any emoji hijinks, can you test your exact same config but using a known working emoji (e.g. try with https://emojipedia.org/party-popper/). If it doesn't work, there might be something else that's wrong with your config.

**Comment by SINE** [comment]
Hello. I edited my initial post to make it clearer.

Yes, I can assure you the popper emoji worked as I tested that myself before posting, it must have to do with the custom emoji.
Also yes, the emoji exists on the server and works being added by the bot to the message as soon as it comes online, but the bot seems unable to identify what role it would belong to when clicked.

My config:
[
  {
    "messageId": "765313699765420064",
    "channelId": "507738749422141440",
    "isUnique": false,
    "emojiRoleMap": {
      ":Redpill:496347789413056512": ["573120759958994946"]
    }
  }
]
